include/camp/resource/host.hpp

```diff
@@ -32,7 +35,7 @@ namespace resources
     class Host
     {
     public:
-      Host(int /* group */ = -1) {}
+      explicit Host(int /* group */ = -1) {}
```

Comment:
Why does this take an arg if it's not used?

Reply:
Consistency, without it the type erasing wrapper wouldn't compile.